Docs/neurons.md

```diff
@@ -161,5 +161,20 @@ neurons:
 
 Here, the first neuron will use the default tts as set in the settings.yml file. The second neuron will use the tts "acapela".
```

Comment:
@Sispheor One way is updating the TTS using the default values defined on settings. The other one is overloading the tts with new parameters ... I guess it makes sense to keep both, so it increases the flexibility. 
What do you think ?